networkx/algorithms/dag.py

```diff
@@ -477,6 +507,36 @@ def is_aperiodic(G):
     NetworkXError
         If `G` is not directed
 
+    Examples
+    --------
+    A graph consisting of one cycle,
+    the length of which is 2
+    (k = 2 divides the length of every cycle in the graph
+    and therefore the graph is not aperiodic)::
+
+        >>> DG = nx.DiGraph([(1, 2), (2, 1)])
+        >>> nx.is_aperiodic(DG)
+        False
+
+    A graph consisting of two cycles,
+    the length of which differs by one
+    (cycle lengths are coprime,
+    so there is no k > 1 dividing each cycle length
```

Comment:
```suggestion
    so there is no single value of k, where k > 1 that divides each cycle length
```
I'm not sure this wording suggestion is any better, but I'm just trying to tie this back into the explanation of `aperiodic` from the docstring summary a bit more. Feel free to ignore!